src/xERC20/contracts/XUltraLRT.sol

```diff
@@ -298,10 +299,10 @@ contract XUltraLRT is
      * @param _priceFeed The address of the price feed
      */
     function setPriceFeed(address _priceFeed) public onlyOwner {
```

Comment:
why is this onlyOwner? should it not be `onlyGov`?

Reply:
owner and gov are same